Dockerfile-build

```diff
@@ -0,0 +1,22 @@
+FROM node:0.10.45
+
+WORKDIR /app
+
+RUN groupadd --gid 10001 app && \
+    useradd --uid 10001 --gid 10001 --home /app --create-home app
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get -y install graphicsmagick && \
+    apt-get clean
+
+USER app
+
+COPY . /app
```

Comment:
1. you should copy ./package.json /app/package.json first 
1. then run `npm install`
1. then copy . /app

This way you get better usage out of the docker image cache if dependencies don't change